src/components/Ideas/IdeaForm.js

```diff
@@ -9,13 +9,14 @@ export const IdeaForm = () => {
     const { artists, getArtists } = useContext(ArtistContext)
     const [idea, setIdea] = useState({
         user_id: "",
-        artistId: "",
+        artistId: "Undecided",
```

Comment:
If Undecided is a possible artist, I'd have this stored in the DB to make it easier to filter on. It would be confusing if an ID could be a string and not a number. We want to keep items in the DB in the same data type as much as possible.